### PR TITLE
feat(appengine): Copy helloworld sample to legacy folder

### PR DESCRIPTION
**Pull request by jahjahbin**
**TLDR**: Create a new GAE Flex folder for dotnet versions 3 and earlier. Copy helloworld sample there as well.

As part of an effort to support .NET 6 in App Engine Flex, we'd like to preserve the samples intended for .NET 3 and earlier. I plan to copy all of the existing samples to an archive first, before eventually migrating the existing samples to .NET 6.

We used this pattern for Go and other langs as well: https://github.com/GoogleCloudPlatform/golang-samples/tree/main/appengine_flexible

As part of this change, I had to update the paths inside `deployTests.ps1` and `runTests.ps1` to point up one extra folder `..`. Otherwise, no functional changes.

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
I've marked as `do not merge` as we discuss these changes internally.

**Comment by amanda-tarafa** [comment]
@jahjahbin Given our internal conversation I think this PR can be closed. Thanks!

**Comment by jahjahbin** [comment]
> @jahjahbin Given our internal conversation I think this PR can be closed. Thanks!

Could you elaborate on the reason? I believe this should be merged -- or polished up after a round of PR review, before merging.

**Comment by amanda-tarafa** [comment]
In general, when we update samples, we don't keep around legacy samples, in particular samples that are targetting out of support frameworks (.NET Core App 2.1 has been out of support since August 2021). Github history will be good enough for the handful of users (if any) that need to consult a basic sample for an out of support framework.

Do you have any strong reasong to keep a legacy samples folder on the repo?

**Comment by jahjahbin** [comment]
> In general, when we update samples, we don't keep around legacy samples, in particular samples that are targetting out of support frameworks (.NET Core App 2.1 has been out of support since August 2021). Github history will be good enough for the handful of users (if any) that need to consult a basic sample for an out of support framework.
> 
> Do you have any strong reasong to keep a legacy samples folder on the repo?

We are planning to keep the legacy sample visible. Check out this page, where we continue to show the "1.15 and earlier" version of a Datastore sample using Go: https://cloud.google.com/appengine/docs/flexible/using-firestore-in-datastore-mode?tab=go#v1.15-and-earlier_1

**Comment by jahjahbin** [comment]
That being said, I remembered something. You'll notice we don't do this for Helloworld in Go, it doesn't refer to the legacy sample. https://cloud.google.com/appengine/docs/flexible/go/create-app.

So, my revised plan is, we copy the necessary examples except Helloworld. Later on, it will be ok to update Helloworld in place. What do you think?

**Comment by amanda-tarafa** [comment]
As per out internal discussion, we can now close this PR. @jahjahbin to confirm and close.
